libbeat/docs/processors-using.asciidoc

```diff
@@ -513,6 +514,45 @@ section.
 NOTE: If you define an empty list of fields under `include_fields`, then only
 the required fields, `@timestamp` and `type`, are exported.
 
+[[rename-fields]]
+=== Rename fields from events
+
+The `rename_fields` processor specifies a list of fields that should be renamed.
+The list of fields to be renamed are listed under `fields`. Each entry contains a `from: old-key` and a `to: new-key` pair. `from` is the origin and `to` the target name of the field.
```

Comment:
Can you please fix the line wrapping again. Also below, the config options lines like `ignore_missing` are long too.

Reply:
Done. I hoped to put this in our editor_config to have it automatically but it's not supported yet. https://github.com/editorconfig/editorconfig/wiki/Property-research:-maximum-line-length

Also pity that IntellJ does not support it based on file types: https://youtrack.jetbrains.com/issue/IDEA-89608